packages/xchain-thorchain/src/client.ts

```diff
@@ -38,6 +38,9 @@ export interface ThorchainClient {
   getDefaultClientUrl(): ClientUrl
   getClientUrlByNetwork(network: Network): string
   setClientUrl(clientUrl: ClientUrl): void
+  getDefaultExplorerUrl(): ExplorerUrl
+  getExplorerUrlByNetwork(network: Network): string
+  setExplorerUrl(explorerUrl: ExplorerUrl): void
```

Comment:
Those methods don't need to be public. Make it private and remove it from interface. Do the same with `getDefaultClientUrl`, `getClientUrlByNetwork`, `setClientUrl`